### PR TITLE
[PRIORITY] MODULARISATION AUDIT PART 1: MAKING SKYRAT GREAT AGAIN

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

Audit completed.

Results:
Fucking shit.

Remedial action:
> Reorganised all modules to conform to the MODULARISATION GUIDELINES.
> Compressed the number of folders.
> Compressed the number of files.
> Removed some redundant and unused files.

Code_4yzxWSK9zI
?????????????????????????

Before:
explorer_H64rHvhOYR

After:
explorer_uqPQoxqhJI

## How This Contributes To The Skyrat Roleplay Experience

The codebase should download about 3.2 seconds quicker.

Things look better.

Things are legal.

## Changelog

:cl:
server: MODULARISATION AUDIT 15/11/21
/:cl:

### CODE REVIEW

**Comment by Gandalf2k15** [comment]
Files that have been deleted have either been merged with another file or were obsolete.

File: modular_skyrat/modules/additionalert/code/pizza/misc.dm [hunk 1]
```diff
@@ -11,3 +11,11 @@
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
 	icon_state = "forensics_red_long"
+
+/obj/item/clothing/under/pizza
+	name = "dogginos employee uniform"
+	desc = "The standard issue for the famous dog founded pizza brand, Dogginos."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
+	icon_state = "dominos"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/additionalert/code/pizza/misc.dm]: This appears to also be in `modular_skyrat/modules/additionalert/code/pizza/misc_clothing.dm`

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Just one tiny change so we don't have duplicate definitions of the same code.

**Comment by ORCACommander** [comment]
*screams in Diff*

File: modular_skyrat/modules/additionalert/code/pizza/misc.dm (hunk 1 above)
Comment (Gandalf2k15) [reply modular_skyrat/modules/additionalert/code/pizza/misc.dm]: Good catch

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**